src/pages/p2p/components/_hero.tsx

```diff
@@ -217,9 +206,27 @@ const Hero = () => {
     const is_rtl = useIsRtl()
     const background = useMemo(() => {
         if (is_tabletL) {
-            return data['p2p_hero_background_mobile']
+            return (
+                <StaticImage
+                    src="../../../images/common/p2p/p2p_hero_background_mobile.png"
+                    alt={localize('_t_p2p_t_')}
+                    formats={['avif']}
+                />
+            )
         } else {
-            return is_rtl ? data['p2p_hero_background_rtl'] : data['p2p_hero_background']
+            return is_rtl ? (
+                <StaticImage
```

Comment:
As only the `src` will be changed, we can do something like this.

```
const imageSrc = is_rtl ? '../../../images/common/p2p/p2p_hero_background_rtl.png' : '../../../images/common/p2p/p2p_hero_background.png';

return (
  <StaticImage
    src={imageSrc}
    alt={localize('_t_p2p_t_')}
    formats={['avif']}
  />
);
```

Reply:
We can't pass dynamic images or variable to `StaticImage`